Userland/Libraries/LibPthread/pthread.cpp

```diff
@@ -143,6 +143,19 @@ void pthread_exit(void* value_ptr)
     exit_thread(value_ptr);
 }
 
+void pthread_cleanup_push(void (*routine)(void *), void *arg)
+{
+    (void)routine;
```

Comment:
The Serenity style for this would be to leave the parameters unnamed in the function signature, I think. Also, should this go in a separate commit, like `LibPthread: add stubs for pthread_cleanup_push/pop`?

Reply:
Aye, you're right. Will get this fixed in a bit. :+1:

Reply:
There we go. :)